src/database/helpers.ts

```diff
@@ -2,11 +2,11 @@ import { TUuid } from '@types';
 
 import { FlowTransducer, GenObject } from './types';
 
-export const withUuid = <T extends {}>(fn: () => TUuid) => (x: T): T => ({ ...x, uuid: fn() });
+export const withUuid = <T extends TObject>(fn: () => TUuid) => (x: T): T => ({ ...x, uuid: fn() });
 
-export const toArray = (object: any): any[] => Object.values(object);
+export const toArray = (obj: any): any[] => Object.values(obj);
```

Comment:
Can we remove `any` here?

Reply:
Changed to `unknown`

Reply:
I wasn't able to find a type fix for this so reverted to renaming.

Reply:
How about

```ts
export const toArray = <T>(obj: T): T[] => Object.values(obj);
```

Reply:
This raises 18 errors.
And the return value is an array of `values` rather than an array of `obj`s.
I tried similar declarations and each raise their own collection of issues:
- `<T>(obj: T): ValuesType<T>[] => Object.values(obj)` 
- `<T>(obj: <Record<string, T>>): T[]`